### PR TITLE
Adding automated testing for pull requests.

### PR DESCRIPTION
**Pull request by jeff-cohere**
It's time to start getting more systematic about changes to the repository, so I've set up some unit tests that automatically run. Code coverage is automatically calculated, so we have an idea of what's being tested and what's not.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: tests
```
Comment (jeff-cohere) [inline .github/workflows/tests.yml:1]: Not a very creative name, is it????

File: ambrs/tests/test_ppe.py [hunk 2]
```diff
@@ -200,8 +200,7 @@ def test_aitken_mode_number_conc_sweep(self):
         self.assertEqual(100, len(ensemble))
         for i, member in enumerate(ensemble):
             step = (math.log10(2e12) - math.log10(3e7)) / 100
-            log_ni = math.log10(3e7) + step
```
Comment (lfierce2) [inline ambrs/tests/test_ppe.py:203]: Laura learns to comment

**Review by lfierce2 — APPROVED** [review]

**Comment by lfierce2** [comment]
Approved and merged. All good?

**Comment by jeff-cohere** [comment]
Yep! Thanks!

**Pull request merged**